bin/cf-blue-green

```diff
@@ -31,13 +31,6 @@ on_fail () {
 }
 
 
-CF_VERSION=$(cf --version)
-# http://stackoverflow.com/a/229606/358804
-if [[ "$CF_VERSION" != *"6.12.0-"* ]] && [[ "$CF_VERSION" != *"6.12.1-"* ]]; then
-  echo "CF CLI version is not compatible. See\n\n\thttps://github.com/18F/cf-blue-green#cf-cli\n"
-  exit 1
-fi
```

Comment:
Doing proper semver parsing in bash seems like more trouble than it's worth...going to make our lives easier and rely on users to RTFM :grimacing: